cmake/macros/macro_deal_ii_setup_target.cmake

```diff
@@ -103,6 +103,13 @@ MACRO(DEAL_II_SETUP_TARGET _target)
     COMPILE_DEFINITIONS "${DEAL_II_USER_DEFINITIONS};${DEAL_II_USER_DEFINITIONS_${_build}}"
     )
 
+  IF (DEAL_II_MSVC)
+    SET_PROPERTY(TARGET ${_target} PROPERTY
+      RUNTIME_OUTPUT_DIRECTORY_${_build} "${CMAKE_CURRENT_BINARY_DIR}"
+      )
+  ENDIF()
```

Comment:
Yes, this is not an elegant solution. Would you prefer we keep the debug/ and release/ directories on windows and adjust the testsuite and buildtests?